models/frax_finance/bnb/frax_finance_bnb_bribes.sql

```diff
@@ -415,58 +472,80 @@ fee_weekly_rewards as (
     group by
         1,
         2
+),
+base_date_with_contracts as (
+    select
+        a.*,
+        b.contract_address,
+        b.contract_name
+    from
+        base_date a
+        cross join c_info_as_per_nader b
 )
 select
     week_start,
     week_end,
     case
         when week_start = cast('2023-01-05' as date) then 0
-        else a.week
+        else week
     end as week,
     a.contract_address,
     a.contract_name,
-    d.bribe as bribe_last_week,
-    round(d.bribe * e.price) as bribe_last_week_usd,
-    b.frax_reserve as start_frax,
-    c.frax_reserve as end_frax,
+    d.Frax_bribe as lwb_frax,
+    round(d.Frax_bribe * i.price) as lwb_frax_usd,
+    b.Frax_reserve as start_frax,
+    c.Frax_reserve as end_frax,
+    round(b.Frax_TVL) as start_frax_TVL,
+    round(c.Frax_TVL) as end_frax_TVL,
     f.fxs_collected_usd,
-    f.fxs_collected,
     round(
-        f.fxs_collected_usd - COALESCE((d.bribe * e.price), 0)
+        coalesce(f.FXS_collected_usd, 0) - (
+            COALESCE((d.FrxETH_bribe * h.price), 0) + COALESCE((d.Frax_bribe * i.price), 0)
+        )
     ) as gross_profit,
-    g.fee_reward_token0,
-    g.fee_reward_token1,
-    b.frxETH_reserve as start_frxETH,
-    c.frxETH_reserve as end_frxETH,
-    round(b.frax_TVL) as start_frax_tvl,
-    round(c.frax_TVL) as end_frax_tvl,
-    round(b.frxETH_TVL) as start_frxETH_tvl,
-    round(c.frxETH_TVL) as end_frxETH_tvl
+    round(g.fee_reward_token0) as fee_reward_token0,
+    round(g.fee_reward_token1) as fee_reward_token1,
+    d.FrxETH_bribe as lwb_frxETH,
+    round(d.FrxETH_bribe * h.price) as lwb_frxETH_usd,
+    b.FrxETH_reserve as start_frxETH,
+    c.FrxETH_reserve as end_frxETH,
+    round(b.FrxETH_TVL) as start_frxETH_TVL,
```

Comment:
it looks like there are still instances throughout, like this section for example

Reply:
I guess its how frxETH is written every where. I haven't capitalized it. 
Cause you won't see frxeth on their website either.

Reply:
its just a spellbook specific database naming standard. for instance, lido uses lowercase on `stETH` to `steth`, such as here:
https://github.com/duneanalytics/spellbook/blob/main/models/lido/liquidity/ethereum/lido_liquidity_ethereum_curve_steth_conc_pool.sql#L28

Reply:
@jeff-dude Noted. 
It's fixed now.